README.md

```diff
@@ -100,7 +100,47 @@ Compile Sass as changes are made:
 
 ### Deployment
 
-See directions in the 18F/openFEC repo.
+##### Likely only useful for 18F team members
+
+Before deploying, install the [Cloud Foundry CLI](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) and the [autopilot plugin](https://github.com/concourse/autopilot):
+
+```
+cf install-plugin autopilot -r CF-Community
+```
+
+To deploy to Cloud Foundry, run `invoke deploy`. The `deploy` task will attempt to detect the appropriate
+Cloud Foundry space based the current branch; to override, pass the optional `--space` flag:
+
+```
+invoke deploy --space dev
+```
+
+The `deploy` task will use the `FEC_CF_USERNAME` and `FEC_CF_PASSWORD` environment variables to log in.
+If these variables are not provided, you will be prompted for your Cloud Foundry credentials.
+
+Credentials for Cloud Foundry applications are managed using user-provided services labeled as
+"fec-creds-prod", "fec-creds-stage", and "fec-creds-dev". Services are used to share credentials between the API and the webapp. To set up a service:
+
+```
+cf target -s dev
+cf cups fec-creds-dev -p '{"SQLA_CONN": "..."}'
+```
+
+To stand up a user-provided credential service that supports both the API and the webapp, ensure that
+the following keys are set:
+
+* SQLA_CONN
+* FEC_WEB_USERNAME
```

Comment:
The username and password variables aren't used anymore. This list of credentials is also different from the lists in the manifest files. Let's drop the comments from the manifest files so we don't have to maintain many sets of credentials and add the missing values here.

Beyond the scope of this patch, but it might be worthwhile to have the app refuse to start if it's missing credentials it needs to run--see https://github.com/18F/openFEC/blob/81aa0d02a1f7248b18df6fff80729a375b15653a/webservices/config.py#L52-L61 for a simple implementation on the API.